src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -245,5 +236,21 @@ public string GetRelativeUrl(Document fileRelativeTo, Document file)
 
             return (file != null ? null : Errors.FileNotFound(relativeTo.ToString(), path), file, null, query, fragment, false, pathToDocset);
         }
+
+        private string ResolveToDocsetRelativePath(string path, Document relativeTo)
+        {
+            var docsetRelativePath = PathUtility.NormalizeFile(Path.Combine(Path.GetDirectoryName(relativeTo.FilePath), path));
+            if (!File.Exists(Path.Combine(relativeTo.Docset.DocsetPath, docsetRelativePath)))
+            {
+                foreach (var (alias, aliasPath) in relativeTo.Docset.ResolveAlias)
```

Comment:
`ResolveAlias` is used as a collection of tuples, not fetching value using the key. Does it need to be `Dictionary`?

Reply:
ok